src/App/Platforms/Android/Autofill/CredentialHelpers.cs

```diff
@@ -77,14 +79,23 @@ private static PublicKeyCredentialCreationOptions GetPublicKeyCredentialCreation
         public static async Task CreateCipherPasskeyAsync(ProviderCreateCredentialRequest getRequest, Activity activity)
         {
             var callingRequest = getRequest?.CallingRequest as CreatePublicKeyCredentialRequest;
+
+            if (callingRequest is null)
```

Comment:
Did you see this happening or it it a _just in case_ check?

Reply:
_Just in case check_ and Google's sample also has it and terminates the operation in case that happens.